src/app/services/xml-parsers/generic-parser.service.ts

```diff
@@ -4,6 +4,7 @@ import { map, scan } from 'rxjs/operators';
 import { NoteComponent } from 'src/app/components/note/note.component';
 import { XMLElement } from 'src/app/models/evt-models';
 import { isNestedInElem, xpath } from 'src/app/utils/dom-utils';
+import { normalizeContentSpaces } from 'src/app/utils/xml-utils';
```

Comment:
Use relative path for imports (I know, there are other wrong imports, we will fix them in the future!)